### PR TITLE
Patch for PedigreeView not reflecting changes to birthday or death

### PR DESCRIPTION
**Pull request by Baizley**
Patch for [bug report 0011366](https://gramps-project.org/bugs/view.php?id=11366&nbn=4#bugnotes).

The PedigreeView was not listening for updates on events, so changes to a birthday or death would not be reflected in the view.

### CODE REVIEW

**Comment by sam-m888** [comment]
@Baizley Looks like you've accidentally pulled in a whole bunch of other commit  can you rebase please?

**Comment by Baizley** [comment]
@sam-m888 it seems like it comes from having branched out from master and the maintenance branch being behind master. I will branch out from the maintenance branch instead and apply my changes.

However, is `maintenance/gramps51` the right target branch? Or should I target master?

**Comment by Nick-Hall** [comment]
This is a bug fix, so _gramps51_ is the correct branch.

File: gramps/plugins/view/pedigreeview.py [hunk 1]
```diff
@@ -784,13 +784,16 @@ def _connect_db_signals(self):
         Connect database signals.
         """
         self._add_db_signal('person-add', self.person_rebuild)
+        # Adding or deleting events will indirectly trigger a person update event
```
Comment (Nick-Hall) [inline gramps/plugins/view/pedigreeview.py]: This comment is slightly confusing here, but you are correct, adding or removing an event will trigger a 'person-update' signal.

Comment (Baizley) [reply gramps/plugins/view/pedigreeview.py]: Where would be the preferred location? Would it be better at the 'event-update' signal or the comment at the top of the function definition?

Comment (Nick-Hall) [reply gramps/plugins/view/pedigreeview.py]: I think that we can just remove the comment.

Comment (Nick-Hall) [reply gramps/plugins/view/pedigreeview.py]: Thanks.  I'll merge this shortly.

**Comment by sam-m888** [comment]
Thank you

**Comment by sam-m888** [comment]
@Nick-Hall Why would the addition of one line in the pedigreeview cause the test on [travis to fail](https://travis-ci.org/gramps-project/gramps/builds/598892899#L2016)?

> FAIL: test_one (gramps.plugins.test.db_undo_and_signals_test.TestMyTestClass_SQLite)
> 
> ----------------------------------------------------------------------
> 
> Traceback (most recent call last):
> 
>   File "/home/travis/build/gramps-project/gramps/gramps/plugins/test/db_undo_and_signals_test.py", line 234, in test_one
> 
>     msg="undo merge families fam1 check")
> 
> nose.proxy.AssertionError: Tuples differ: ('000[61 chars]2', [], (3, ''), [], [], [], [], [], [], 1571265344, [], False) != ('000[61 chars]2', [], (3, ''), [], [], [], [], [], [], 1571265345, [], False)
> 
> First differing element 12:
> 
> 1571265344
> 
> 1571265345
> 
>   ('0000000500000005',
> 
>    'F0000',
> 
>    '0000000100000001',
> 
>    '0000000200000002',
> 
>    [],
> 
>    (3, ''),
> 
>    [],
> 
>    [],
> 
>    [],
> 
>    [],
> 
>    [],
> 
>    [],
> 
> -  1571265344,
> 
> ?           ^
> 
> +  1571265345,
> 
> ?           ^
> 
>    [],
> 
>    False) : undo merge families fam1 check
> 
> -------------------- >> begin captured stdout << ---------------------
> 
> ('person-add', (['0000000100000001', '0000000200000002', '0000000300000003', '0000000400000004'],)),
> 
> ('family-add', (['0000000500000005', '0000000600000006'],)),
> 
> ('person-update', (['0000000100000001', '0000000200000002', '0000000300000003', '0000000400000004'],)),
> 
> ('person-delete', (['0000000300000003', '0000000400000004'],)),
> 
> ('family-delete', (['0000000600000006'],)),
> 
> ('person-update', (['0000000100000001', '0000000200000002', '0000000100000001', '0000000200000002'],)),
> 
> ('family-update', (['0000000500000005', '0000000500000005'],)),
> 
> ('note-delete', (['0000000700000007'],)),
> 
> ('note-delete', (['0000000700000007'],)),
> 
> ('person-add', (['0000000400000004', '0000000300000003'],)),
> 
> ('family-add', (['0000000600000006'],)),
> 
> ('person-update', (['0000000200000002', '0000000100000001', '0000000200000002', '0000000100000001'],)),
> 
> ('family-update', (['0000000500000005', '0000000600000006', '0000000600000006', '0000000500000005'],)),
> 
> --------------------- >> end captured stdout << ----------------------

**Comment by sam-m888** [comment]
@Nick-Hall Hmm had travis re-run the test and it worked!

**Comment by prculley** [comment]
I may have mentioned the Travis failure in another PR; the change time on the elements that differ is one second different.  This means that in the several milliseconds that it took to create the elements to be compared, the clock ticked over the second mark.  A small chance, but a real possibility.  The test needs to be reworked to mock up the db time functions so that the clock doesn't advance.  Until then we will get these once in a while.

**Comment by sam-m888** [comment]
I believe I recall reading that message. Thanks for the reminder :)

**Pull request merged**